ads/_config.js

```diff
@@ -938,6 +938,10 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'remixd': {
+    preconnect: 'https://tags.remixd.com',
```

Comment:
Did you have a chance to look at implementing [`renderStart`](https://github.com/ampproject/amphtml/blob/73f4d0eaa7a217ecac0dd4c6181dc4f9297766ef/ads/README.md#available-apis)?  It should give you some performance gains, but ultimately it is your decision?

Reply:
@glendza Looks like this is an open question for you.

Reply:
Done